### PR TITLE
implement 2021 social insurance changes

### PR DESCRIPTION
**Pull request by Eric-Sommer**
- Higher average contribution rate for statutory health insurance
- Higher Assessment ceilings
- Update of pension-relevant parameters
- correct one typo

### CODE REVIEW

File: gettsim/parameters/ges_renten_vers.yaml [hunk 1]
```diff
@@ -228,9 +228,13 @@ beitragspflichtiger_durchschnittslohn:
...
     scalar: 38212
   2019-01-01:
-    scalar: 38901
+    scalar: 39301
+    reference: §1 (1) Sozialversicherungs-Rechengrößenverordnung 2021
   2020-01-01:
     scalar: 40551
+  2021-01-01:
+    scalar: 41541
+    reference: https://www.bundesregierung.de/breg-de/aktuelles/beitragsbemessungsgrenzen-2021-1796480
```
Comment (mjbloemer) [inline gettsim/parameters/ges_renten_vers.yaml:237]: suggestion:

```
2018-01-01:
  scalar: 38212
  reference: V. v. 17.12.2019 BGBl. I S. 2848.
2019-01-01:
  scalar: 39301
  reference: V. v. 30.11.2020 BGBl. I S. 2612.
2020-01-01:
  scalar: 40551
  reference: V. v. 17.12.2019 BGBl. I S. 2848.
  note: Das vorläufige Durchschnittsentgelt.
2021-01-01:
  scalar: 41541
  reference: V. v. 30.11.2020 BGBl. I S. 2612.
  note: Das vorläufige Durchschnittsentgelt.
```

File: gettsim/parameters/soz_vers_beitr.yaml [hunk 2]
```diff
@@ -488,6 +495,15 @@ beitr_bemess_grenze:
...
       ost: 6450
     reference: V. v. 17.12.2019 BGBl. I S. 2848
+  2021-01-01:
+    ges_krankenv:
+      west: 4837.5
+      ost: 4837.5
+    rentenv:
+      west: 7100
+      ost: 6700
+    note: §3, §4 Sozialversicherungs-Rechengrößenverordnung 2021, V. v. 30.11.2020, BGBl. I S. 2612
```
Comment (mjbloemer) [inline gettsim/parameters/soz_vers_beitr.yaml:505]:
```suggestion
    reference: V. v. 30.11.2020 BGBl. I S. 2612.
```

File: gettsim/parameters/soz_vers_beitr.yaml [hunk 3]
```diff
@@ -605,6 +621,10 @@ bezugsgröße:
     reference: §2 V. v. 17.12.2019 BGBl. I S. 2848
     west: 3185
     ost: 3010
+  2021-01-01:
+    west: 3290
+    ost: 3115
+    note: §2 Sozialversicherungs-Rechengrößenverordnung 2021, V. v. 30.11.2020, BGBl. I S. 2612
```
Comment (mjbloemer) [inline gettsim/parameters/soz_vers_beitr.yaml:627]:
```suggestion
    reference: V. v. 30.11.2020 BGBl. I S. 2612.
```

**Review by mjbloemer — CHANGES_REQUESTED** [review]
I suggest a few minor reference style changes.

File: gettsim/parameters/soz_vers_beitr.yaml (hunk 3 above)
Comment (Eric-Sommer) [reply gettsim/parameters/soz_vers_beitr.yaml:627]: OK. I left the `§2` in, though.

File: gettsim/parameters/ges_renten_vers.yaml [hunk 4]
```diff
@@ -226,11 +226,19 @@ beitragspflichtiger_durchschnittslohn:
   2017-01-01:
     scalar: 37077
   2018-01-01:
-    scalar: 38212
+   scalar: 38212
+   reference: V. v. 17.12.2019 BGBl. I S. 2848.
```
Comment (mjbloemer) [inline gettsim/parameters/ges_renten_vers.yaml:230]: still incorrect indentation?

**Review by mjbloemer — CHANGES_REQUESTED** [review]

File: gettsim/parameters/ges_renten_vers.yaml (hunk 4 above)
Comment (Eric-Sommer) [reply gettsim/parameters/ges_renten_vers.yaml:230]: corrected now...frohes neues ;)

**Review by mjbloemer — APPROVED** [review]

**Pull request merged**